src/js/pages/Index.js

```diff
@@ -72,6 +77,9 @@ var Index = React.createClass({
 
     ConfigStore.fetchCCID();
     ConfigStore.addChangeListener(EventTypes.CONFIG_ERROR, this.onConfigError);
+    compare.subscribe(values => {
+      showNotification(getVersionFromVersionObject(values[1]));
```

Comment:
shall we `map` filtered result so that we could 
```js
compare.subscribe(newVersion => showNotification(newVersion))
```
?

Reply:
Good point, done.